### DISCUSSION TITLE
fabric and scheduler

### DISCUSSION THREAD

**Discussion by pfeatherstone** [Lightning-AI/pytorch-lightning#18359] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule] [answered]
How do I use fabric with a scheduler. The fabric API doesn't seem to have a `fabric.setup_scheduler()` function.
Thanks

**Comment by awaelchli** [comment, marked as answer]
@pfeatherstone There is no special need to setup the scheduler. You can just use it side by side with your optimizer and step it normally.
